runtime-sdk/modules/rofl-market/src/policy.rs

```diff
@@ -0,0 +1,123 @@
+use std::collections::BTreeMap;
+
+use base64::prelude::*;
+
+use oasis_runtime_sdk::{
+    core::{
+        common::crypto::signature::{PublicKey, Signature},
+        consensus::registry::{EndorsedCapabilityTEE, Node},
+        host::attestation::{LabelAttestation, ATTEST_LABELS_SIGNATURE_CONTEXT},
+    },
+    modules::rofl::{
+        policy::{AllowedEndorsement, EndorsementPolicyEvaluator},
+        Error,
+    },
+    types::address::Address,
+    Context,
+};
+
+use crate::{state, types::InstanceId};
+
+/// Name of the ROFL app instance metadata key used to store the provider attestation.
+pub const METADATA_KEY_POLICY_PROVIDER_ATTESTATION: &str = "net.oasis.policy.provider";
```

Comment:
Why is this defined twice?
Why is `KEY_POLICY`  in the name?

Reply:
See #2265.